include/wlr/types/wlr_output_layout.h

```diff
@@ -4,14 +4,20 @@
 #include <wayland-util.h>
 #include <stdbool.h>
 
+struct wlr_output_layout_state;
+
 struct wlr_output_layout {
     struct wl_list outputs;
+	struct wlr_output_layout_state *state;
```

Comment:
You're not to blame now but doesn't match previous indentation, I'd say fix that while you're here... Mixed indentation is evil.